file_picker/uploads/file_pickers.py

```diff
@@ -14,7 +16,8 @@ class Meta(object):
 
     def save(self, commit=True):
         image = super(ImageForm, self).save(commit=False)
-        file_path = self.cleaned_data['file']
+        # Strip any directory names from the filename
+        file_path = os.path.basename(self.cleaned_data['file'])
```

Comment:
Oh, I just noticed that our tests do cover the identical code change in forms.py (lines 52-53). If I revert that change, then the tests pass on Django 1.8 and 1.9, but fail on 1.10.